README.md

```diff
@@ -6,4 +6,13 @@ This is the Dockerfile (etc.) used for building the image that runs python scrip
 
 Because the build process of this image has become quite lengthy, we've split it into three. The base images are at [here](https://github.com/Kaggle/docker-python0) and [here](https://github.com/Kaggle/docker-python1).
 
+**To start container [Learn more](http://blog.kaggle.com/2016/02/05/how-to-get-started-with-data-science-in-containers/comment-page-1/)
+
+Mac
```

Comment:
No, more that the linked blogpost explains this stuff in enough detail, and I don't feel we need to reproduce the advice here…?